docs/src/loop.rst

```diff
@@ -218,3 +234,8 @@ API
        Any previous value returned from :c:func`uv_backend_fd` is now
        invalid. That function must be called again to determine the
        correct backend file descriptor.
+
+.. c:function:: uv_loop_stats_t* uv_loop_stats(uv_loop_t* loop)
```

Comment:
+1 ... was wanting to avoid the copying if at all possible and went back and forth on this. Copying works.